src/Interface.jl

```diff
@@ -532,6 +532,10 @@ end
 function evaluate(ri::FragmentBasedReactiveFilmGrowthInterfaceConstantT, dydt, domainfilm, domain2, Vfilm, cstot)
     kfs, krevs = getkfskrevs(ri)
     addreactionratecontributions!(dydt, ri.fragmentbasedrxnarray, ri.rxnarray, cstot, kfs, krevs, Vfilm, domainfilm.indexes[3], ri.Mws, domainfilm.indexes[1]:domainfilm.indexes[2])
+    if hasproperty(ri.domain2, :epsilon)
+        epsilon = ri.domain2.epsilon
+        dydt[ri.domain2.indexes[3]] = dydt[domainfilm.indexes[3]] / domainfilm.rho / (1 - epsilon) * epsilon
```

Comment:
This is hard coded for .indexes[3] being the particular thermo variable you want. You need to extract the index of the specific thermo variable you want.